### ISSUE TITLE
Add isort check to CI

### ISSUE THREAD

**Issue by WeilerP** [theislab/scvelo#375]
## Description

The CI needs to include a check ensuring imports are ordered according to `isort` format. See [here](https://pycqa.github.io/isort/docs/configuration/github_action/) for isort's demo on how to achieve this.

**Comment by WeilerP** [comment]
Prior to adding `isort` to the CI, all libraries used within the code base should be installed through `pip install -r requirements-dev.txt`.

**Comment by Zethson** [comment]
Maybe it would be a good idea to just add a job for pre-commit.
Then you would just need to add your linting tools to pre-commit and don't need to adapt the CI every time.

**Comment by WeilerP** [comment]
Yes, was going to do that in the future. Should probably do it right away instead of this issue.
